tests/unit/amo/sagas/test_collections.js

```diff
@@ -523,13 +524,19 @@ describe(__filename, () => {
     });
 
     describe('create logic', () => {
+      const params = {
+        description: { 'en-US': 'Collection description' },
```

Comment:
The problem is that we need to get `state` via `sagaTester.getState()`, and `sagaTester` isn't available here because it is created in `beforeEach`. So I'd need to move a bunch of stuff around and/or introduce some duplication in order to be able to access `state` during this setup process, which I think will just make the whole thing more complicated. So I'd prefer to leave it as it is.

Reply:
ah, usually we transform the variable into a function like `getParams()`

Reply:
I'm not following how that would address the issue I described.

Reply:
```js
const getParams = ({ lang }) => {

getParams({ lang: state.api.lang })
```

Reply:
Because `collectionDetailResponse` also needs access to `params`, doing this still requires me to duplicate a bit of code, but I did it anyway because it's only one statement duplicated. Thanks for the advice. 👍